src/test/ui/regions/regions-static-bound.ll.nll.stderr

```diff
@@ -33,14 +33,35 @@ LL |     static_id(&u); //[ll]~ ERROR explicit lifetime required in the type of
    |     ^^^^^^^^^^^^^ lifetime `'static` required
 
 error[E0621]: explicit lifetime required in the type of `v`
-  --> $DIR/regions-static-bound.rs:28:5
+  --> $DIR/regions-static-bound.rs:29:5
    |
 LL | fn error(u: &(), v: &()) {
    |                     --- help: add explicit lifetime `'static` to the type of `v`: `&'static ()`
 ...
 LL |     static_id_indirect(&v); //[ll]~ ERROR explicit lifetime required in the type of `v` [E0621]
    |     ^^^^^^^^^^^^^^^^^^^^^^ lifetime `'static` required
 
-error: aborting due to 3 previous errors
+error: unsatisfied lifetime constraints
+  --> $DIR/regions-static-bound.rs:25:5
+   |
+LL | fn error(u: &(), v: &()) {
+   |             -       - let's call the lifetime of this reference `'2`
+   |             |
+   |             let's call the lifetime of this reference `'1`
+LL |     static_id(&u); //[ll]~ ERROR explicit lifetime required in the type of `u` [E0621]
+   |     ^^^^^^^^^^^^^ argument requires that `'1` must outlive `'2`
```

Comment:
Curious... but I *suspect* these "extra errors" arise from the duplicate edges being yielded up by `Edges`